faunadb-scala/src/main/scala/faunadb/values/Codec.scala

```diff
@@ -145,11 +198,12 @@ object Decoder {
       }
   }
 
-  implicit object ByteArrayDecoder extends Decoder[Array[Byte]] {
+  implicit def EitherDecoder[A, B](implicit a: Decoder[A], b: Decoder[B], tagA: ClassTag[A], tagB: ClassTag[B]): Decoder[Either[A, B]] = new Decoder[Either[A, B]] {
```

Comment:
one way to get around that would be to push any requirements for error handling into the generic decoder objects themselves, so that Decoder[A] and Decoder[B] are the only implicit parameters.